vm/src/stdlib_natives.cpp

```diff
@@ -310,6 +311,20 @@ static void program_arguments(OperandStack &caller_stack, runtime_memory &mem) {
     }
 }
 
+static void process_wait(OperandStack &caller_stack, runtime_memory &) {
+    pid_t pid = static_cast<pid_t>(caller_stack.pop_int());
+    int status;
+    if (waitpid(pid, &status, 0) == -1) {
+        throw RuntimeException("Failed to wait for process " + std::to_string(pid) + ": " + strerror(errno) + ".");
+    }
+}
+
+static void process_wait_all(OperandStack &, runtime_memory &) {
+    int status;
+    while (wait(&status) > 0)
+        ;
```

Comment:
```suggestion
    while (wait(&status) > 0);
```

Reply:
Are you referring to `// clang-format off`? This is the format for `while` statements that don't have bodies.

Reply:
its disgusting (not opinionated)

Reply:
To kill two birds with one stone, a *clang-tidy* check may also be suppressed here by declaring an empty body. *clang-format* creates:
```c
    while (wait(&status) > 0) {
    }
```